app/models/application_letter.rb

```diff
@@ -98,6 +123,14 @@ def status_cannot_be_changed
     end
   end
 
+  # Validator for status_change_allowed?
+  # Adds error
+  def status_notification_sent_cannot_be_changed
+    unless status_notification_sent_change_allowed?
+      errors.add(:event, "Das Status-Benachrichtungsflag kann noch nicht gesetzt werden") #TODO
```

Comment:
Is that really needed for this PR? There is another non-localized string as error message in this file, so it needs to be fixed (later) anyway, so I found no example and the things i tried didnt work.